app/src/main/java/tech/zafrani/companionforpubg/activities/DrawerActivity.java

```diff
@@ -105,9 +94,6 @@ public boolean onNavigationItemSelected(@NonNull final MenuItem item) {
 
     //region methods
     private void mapSelected() {
-        if (this.contentLayout == null) {
```

Comment:
If the activity is destroyed in the middle of this function being called, the unbind method will set these views to null. It will be hard (nearly impossible) to trigger in it's current state but any future delayed callbacks or animations will trigger it.